src/views/home/index.js

```diff
@@ -35,10 +35,20 @@ export const Home = () => {
     setRight(nextRight.message)
   }
 
+  const handleArrowKeyDown = async (e) => {
+    if (e.keyCode === 37) {
```

Comment:
@wood-crafter use `e.key === 'ArrowUp'` here, instead of `e.keyCode === 37`

You can try it out here...!
https://www.toptal.com/developers/keycode

Same for other places